content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -275,14 +327,18 @@ order or an invoice.
 Workflows
 =========
 
-This section provides an overview of the actions that trigger API calls for tax computation.
+This section provides an overview of the actions that trigger API calls for tax computation, and how
```

Comment:
you might want to explain what API calls in a couple of words or only add a link to an explanation.